tests/dags/providers/provider_api_scripts/test_museum_victoria.py

```diff
@@ -88,6 +89,16 @@ def test_get_record_data():
     assert actual_image_data[0] == expected_image_data
 
 
+def test_filetype_gets_added_by_image_store():
+    media = _get_resource_json("record_data.json")
+    with patch.object(mv.media_stores["image"], "save_item") as mock_save:
+        mv.process_batch([media])
+
+    actual_image = mock_save.call_args[0][0]
+
+    assert "jpg" == actual_image.filetype
```

Comment:
Gah, I'm so sorry - I keep forgetting that this is effectively covered by the media store class and probably doesn't need to be tested at the ingester level! I'll still try to figure out why this test is failing in certain cases, but I think barring any weird discoveries there we can probably drop this test.